kahuna/public/js/crop/controller.js

```diff
@@ -1,236 +1,209 @@
 import angular from 'angular';
 
 import '../components/gr-keyboard-shortcut/gr-keyboard-shortcut';
-
-var crop = angular.module('kahuna.crop.controller', ['gr.keyboardShortcut']);
-
-crop.controller('ImageCropCtrl',
-                ['$scope', '$rootScope', '$stateParams', '$state',
-                 '$filter', '$document', 'mediaApi', 'mediaCropper',
-                 'image', 'optimisedImageUri', 'keyboardShortcut', 'storage',
-                 function($scope, $rootScope, $stateParams, $state,
-                          $filter, $document, mediaApi, mediaCropper,
-                          image, optimisedImageUri, keyboardShortcut, storage) {
-
-    const ctrl = this;
-    const imageId = $stateParams.imageId;
-    if ($stateParams.cropType) {
-        storage.setJs('cropType', $stateParams.cropType, true);
-    }
-    ctrl.cropType = storage.getJs('cropType', true);
-
-    keyboardShortcut.bindTo($scope)
```

Comment:
moved to bottom of file because its not that interesting

spoiler: its fewer LOC too!